rpm_spec/securedrop-workstation.spec

```diff
@@ -0,0 +1,63 @@
+Name:		securedrop-workstation
+Version:	0.0.1
+Release:	1%{?dist}
+Summary:	SecurdDrop Workstation
+
+Group:		Library
+License:	GPLv3+
```

Comment:
We're actually using AGPLv3 for this project.

Reply:
Making the change.